tools/probes/probes_main.c

```diff
@@ -234,16 +245,17 @@ void parse_data(char *file_in)
 		fclose(fd_in);
 		exit(0);
 	}
-	memset(&data, 0, sizeof(uint32_t) * DATA_READ_LIMIT);
+	memset(&data, 0, DATA_READ_LIMIT);
 	memset(&files, 0, sizeof(struct wave_files) * FILES_LIMIT);
 
 	/* data read loop to process DATA_READ_LIMIT bytes at each iteration */
 	do {
-		i = fread(&data, sizeof(uint32_t), DATA_READ_LIMIT, fd_in);
+		i = fread(&data, 1, DATA_READ_LIMIT, fd_in);
```

Comment:
same here - and subjectively `fread(data,...)` seems clearer to me, but it's just me